Voog/design-dorpat
# Add sale price support

Adds sale price support, which includes:

- Adding sale prices to auto-rendered product pages
- Making sale prices react to dark backgrounds
- Making sale prices react to design editor changes

Closes #26 

Testing:
- Make sure that sale prices are displayed on auto-rendered product pages
- Make sure that sale prices react to design editor changes
- Make sure that sale prices react to dark background
